ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -4,10 +4,27 @@ classes:
     DEFAULTS:
         base: [zenpacklib.Component]
 
+    HardDisk:
```

Comment:
There's a Products.ZenModel.HardDisk.HardDisk class. It'd probably make more sense to use that or subclass it if needed like has been done for FileSystem.